cmd/validate-krew-manifest/main.go

```diff
@@ -77,6 +77,16 @@ func validateManifestFile(path string) error {
 	}
 	glog.Infof("structural validation OK")
 
+	if maxLen := 80; !validateLineLength(p.Spec.Description, maxLen) {
+		return fmt.Errorf("line length in `description` exceeds %d characters", maxLen)
+	}
+	if maxLen := 70; !validateLineLength(p.Spec.ShortDescription, maxLen) {
+		return fmt.Errorf("line length in `shortDescription` exceeds %d characters", maxLen)
+	}
+	if maxLen := 80; !validateLineLength(p.Spec.Caveats, maxLen) {
```

Comment:
Yes, I was also considering to look for `http(s)?...` and ignore those lines for length validation. This may be the better option, but it's also a little more complex.